Microsoft.Toolkit.Uwp.UI.Controls.Layout/GridSplitter/GridSplitter.Events.cs

```diff
@@ -16,54 +16,20 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// </summary>
     public partial class GridSplitter
     {
-        // Symbols for GripperBar in Segoe MDL2 Assets
-        private const string GripperBarVertical = "\xE784";
-        private const string GripperBarHorizontal = "\xE76F";
-        private const string GripperDisplayFont = "Segoe MDL2 Assets";
-
         private void GridSplitter_Loaded(object sender, RoutedEventArgs e)
         {
             _resizeDirection = GetResizeDirection();
             _resizeBehavior = GetResizeBehavior();
 
-            // Adding Grip to Grid Splitter
-            if (Element == default(UIElement))
-            {
-                CreateGripperDisplay();
-                Element = _gripperDisplay;
-            }
+            ManipulationMode = ManipulationModes.TranslateX | ManipulationModes.TranslateY;
 
-            if (_hoverWrapper == null)
+            // Adding Grip to Grid Splitter
+            if (Content == null)
             {
-                var hoverWrapper = new GripperHoverWrapper(
-                    CursorBehavior == SplitterCursorBehavior.ChangeOnSplitterHover
-                    ? this
-                    : Element,
-                    _resizeDirection,
-                    GripperCursor,
-                    GripperCustomCursorResource);
-                ManipulationStarted += hoverWrapper.SplitterManipulationStarted;
-                ManipulationCompleted += hoverWrapper.SplitterManipulationCompleted;
-
-                _hoverWrapper = hoverWrapper;
-            }
-        }
+                Content =
+                    _resizeDirection == GridResizeDirection.Columns ? GripperBarVertical : GripperBarHorizontal;
 
-        private void CreateGripperDisplay()
-        {
-            if (_gripperDisplay == null)
-            {
-                _gripperDisplay = new TextBlock
-                {
-                    FontFamily = new FontFamily(GripperDisplayFont),
-                    HorizontalAlignment = HorizontalAlignment.Center,
-                    VerticalAlignment = VerticalAlignment.Center,
-                    Foreground = GripperForeground,
-                    Text = _resizeDirection == GridResizeDirection.Columns ? GripperBarVertical : GripperBarHorizontal
-                };
-                _gripperDisplay.SetValue(
-                    Windows.UI.Xaml.Automation.AutomationProperties.AccessibilityViewProperty,
-                    Windows.UI.Xaml.Automation.Peers.AccessibilityView.Raw);
+                GripperCursor = _resizeDirection == GridResizeDirection.Columns ? CoreCursorType.SizeWestEast : CoreCursorType.SizeNorthSouth;
```

Comment:
Wouldn't we still want the gripper cursor even if the content is set by the dev?

Reply:
Indeed - change made.